lib/googleanalytics/index.js

```diff
@@ -4,7 +4,7 @@ const googleanalytics = ($, options) => {
   if (options.gaConfigPath) {
     const config = utils.getRelativeFile(options.gaConfigPath, options.cwd)
     const $script = $('<script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js" />')
-    const $config = $(`<amp-analytics type="googleanalytics"/><script type="application/json">${config}</script></amp-analytics>`)
+    const $config = $(`<amp-analytics type=${options.vendor}><script type="application/json">${config}</script></amp-analytics>`)
```

Comment:
IMO, `vender` option might be ambiguous. 🤔  User may think what kind of vender it is.

How about `analyticsVendor` ?